packages/bitcore-node/package.json

```diff
@@ -44,7 +44,8 @@
     "lint:style": "prettier --check 'src/**/*.ts'",
     "fix": "npm run fix:style && npm run fix:lint",
     "fix:lint": "tslint --fix -c ../../tslint.json 'src/**/*.ts'",
-    "fix:style": "prettier --write 'src/**/*.ts'"
+    "fix:style": "prettier --write 'src/**/*.ts'",
+    "migration": "cd migrations && node fixUnspentInputs.js --network regtest --dryrun false"
```

Comment:
Given that this is a one-time run, we probably don't want to have this as a script

Reply:
Haha my bad. Meant to remove this. Just used for development. I will remove. (gosh, I need to get better about this)